src/node/rpc/memberfrontend.h

```diff
@@ -72,22 +72,43 @@ namespace ccf
         // add a new member
         {"new_member",
          [this](Store::Tx& tx, const nlohmann::json& args) {
-           const Cert cert = args;
-           auto mc = tx.get_view(this->network.member_certs);
+           const Cert pem_cert = args;
+           auto [mc, m, ack, v] = tx.get_view(
+             this->network.member_certs,
+             this->network.members,
+             this->network.member_acks,
+             this->network.values);
            // the cert needs to be unique
+           auto cert = tls::make_verifier(pem_cert)->raw_cert_data();
```

Comment:
I think we should rename `raw_cert_data()` to `der_cert_data()`. This looks like casting some mbedtls internal details, and we need to look at the docs to work out this is actually in DER format.